### DISCUSSION TITLE
Media player pattern?

### DISCUSSION THREAD

**Discussion by QiiFeng** [twilio-labs/paste#2339] [General] [answered]
Hey team, I noticed that the Media player is one of the most requested patterns and is open for contribution. Just realized Twilio Intelligence Team which I am working for has a relevant use case. It is about the audio player (pls see below). I am thinking this could be sth potentially I can contribute to. But I am wondering if this is something the team is looking for from your roadmap side?

Thanks 

<img width="1077" alt="Screenshot 2022-04-04 at 17 53 26" src="https://user-images.githubusercontent.com/86003634/161593552-3ea7a681-2036-45f5-9bd3-df1ded606d8c.png">

**Comment by andioneto** [comment]
Hello @QiiFeng 👋

I will chat with the team and get back to you with more information in the next few days. In the meantime, we generally welcome contributions! Please checkout these resources that provide a more general overview about contributions for patterns and components:

- [Paste pattern contributions](https://paste.twilio.design/introduction/contributing/patterns)
- [Paste component contributions](https://paste.twilio.design/introduction/contributing/components)

**Comment by andioneto** [comment, marked as answer]
Hi @QiiFeng 👋 

Unfortunately we are not accepting contributions until July 2022. Our design team is growing and until we finish onboarding new folks we have limited bandwidth. If it works for your timeline, feel free to check in with us in July.

Thanks for you interest and taking the time to help us improve Paste! 🏅

**Comment by QiiFeng** [comment]
Hi Andi,

Thanks for getting back to me. Will reach out to your team in July.

On Tue, Apr 5, 2022 at 5:54 PM Andi Oneto ***@***.***> wrote:

> Hi @QiiFeng
> <https://urldefense.com/v3/__https://github.com/QiiFeng__;!!NCc8flgU!dzYrx9Mx5M0Tq6wId8p6DW3E2YhzA1CTfhZpb6MoDpnE2V3zMK7lcZQ-sh78J1bHL5sqbnLGLQNIMUe2DdtLgmM$>
> 👋
>
> Unfortunately we are not accepting contributions until July 2022. Our
> design team is growing and until we finish onboarding new folks we have
> limited bandwidth. If it works for your timeline, feel free to check in
> with us in July.
>
> Thanks for you interest and taking the time to help us improve Paste! 🏅
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/twilio-labs/paste/discussions/2339*discussioncomment-2509794__;Iw!!NCc8flgU!dzYrx9Mx5M0Tq6wId8p6DW3E2YhzA1CTfhZpb6MoDpnE2V3zMK7lcZQ-sh78J1bHL5sqbnLGLQNIMUe2oiGj74E$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AUQE7MSOGBGRXJ7YNNO7WDLVDRV55ANCNFSM5SQFHV5A__;!!NCc8flgU!dzYrx9Mx5M0Tq6wId8p6DW3E2YhzA1CTfhZpb6MoDpnE2V3zMK7lcZQ-sh78J1bHL5sqbnLGLQNIMUe2FlskiP4$>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Qi Feng
Product Designer
[image: Twilio]
EMAIL  ***@***.***
